bin2c/bin2c.c

```diff
@@ -44,19 +47,34 @@
     }
 #endif
 
+	unsigned char * fprintList(FILE *outputFile, unsigned char *bufptr, unsigned char len, const char* formatStr)
```

Comment:
Indentation and code style does not match here - please match the rest of the file.

Reply:
I fixed the indentation, but what do you mean by code style, which rule am I violating?

Reply:
Oh, you mean variable declarations at the begining of the function?